File: MVC_DiceGame/MVC_DiceGame_csharp/DiceGame/controller/Player.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DiceGame.controller
{
	class Player
	{
		public void PlayGame(model.DiceGame a_game, view.Console a_view)
		{
			a_view.DisplayInstructions();

			while (a_view.WantsToPlay())
			{
				a_view.DisplayInstructions();

				a_view.DisplayResult(a_game.Play(), a_game.GetDice1Value(), a_game.GetDice2Value());
				
			}

		}
	}
}

File: MVC_DiceGame/MVC_DiceGame_ruby/controller/Player.rb
module Controller
  class Player

    def playGame(a_game, a_view)
      a_view.display_instructions
      until a_view.wants_to_quit? do
        a_view.display_instructions
        a_view.display_result(a_game.play, a_game.dice_one_value, a_game.dice_two_value)
      end
    end

  end
end
